storybook/pages/TokenSelectorViewPage.qml

```diff
@@ -0,0 +1,204 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core.Utils 0.1
+
+import Storybook 1.0
+import Models 1.0
+
+import SortFilterProxyModel 0.2
+
+import AppLayouts.Wallet.views 1.0
+import AppLayouts.Wallet.stores 1.0
+import AppLayouts.Wallet.adaptors 1.0
+
+import shared.stores 1.0
+import utils 1.0
+
+SplitView {
```

Comment:
The storybook page, similarly as unit test, should be focused on a single "unit" of code, in this case `TokensSelectorView`. Therefore is shouldn't test or depend on correctness of other components like adaptor, which should be fully independent. I think the good idea is to have separate pages for views and adaptors. We already have some utility storybook components to easily display generic lists, they can be used to visualize input and output of adaptor withing a dedicated pages. Such page should contain controls for switches exposed from the adaptor.

Later, while progressing with refactors we will have sb pages combining both views and stores under more high-level components.